## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 462971`
Commit: `e74ea94ba607daa2f73c233726cd9ae7684f2219`

---
### `x86_64-linux`
<details>
  <summary>:x: 44 packages failed to build:</summary>
  <ul>
    <li>ants</li>
    <li>apacheHttpdPackages.mod_tile (apacheHttpdPackages_2_4.mod_tile)</li>
    <li>cloudcompare</li>
    <li>gplates</li>
    <li>merkaartor</li>
    <li>mrtrix</li>
    <li>otb</li>
    <li>python312Packages.medpy</li>
    <li>python312Packages.medpy.dist</li>
    <li>python312Packages.medvol</li>
    <li>python312Packages.medvol.dist</li>
    <li>python312Packages.napari-nifti</li>
    <li>python312Packages.napari-nifti.dist</li>
    <li>python312Packages.pymedio</li>
    <li>python312Packages.pymedio.dist</li>
    <li>python312Packages.pyotb</li>
    <li>python312Packages.pyotb.dist</li>
    <li>python312Packages.pyradiomics</li>
    <li>python312Packages.pyradiomics.dist</li>
    <li>python312Packages.simpleitk</li>
    <li>python312Packages.simpleitk.dist</li>
    <li>python312Packages.torchio</li>
    <li>python312Packages.torchio.dist</li>
    <li>python313Packages.medpy</li>
    <li>python313Packages.medpy.dist</li>
    <li>python313Packages.medvol</li>
    <li>python313Packages.medvol.dist</li>
    <li>python313Packages.napari-nifti</li>
    <li>python313Packages.napari-nifti.dist</li>
    <li>python313Packages.pymedio</li>
    <li>python313Packages.pymedio.dist</li>
    <li>python313Packages.pyotb</li>
    <li>python313Packages.pyotb.dist</li>
    <li>python313Packages.pyradiomics</li>
    <li>python313Packages.pyradiomics.dist</li>
    <li>python313Packages.simpleitk</li>
    <li>python313Packages.simpleitk.dist</li>
    <li>python313Packages.torchio</li>
    <li>python313Packages.torchio.dist</li>
    <li>simpleitk</li>
    <li>sumo</li>
    <li>umap</li>
    <li>umap.dist</li>
    <li>vtk-dicom</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 263 packages built:</summary>
  <ul>
    <li>appcsxcad</li>
    <li>c3d</li>
    <li>csxcad</li>
    <li>elastix</li>
    <li>elmerfem</li>
    <li>entwine</li>
    <li>exhibit</li>
    <li>ezminc</li>
    <li>f3d</li>
    <li>f3d.man</li>
    <li>freecad</li>
    <li>froide</li>
    <li>froide-govplan</li>
    <li>froide-govplan.dist</li>
    <li>froide.dist</li>
    <li>gdal (python313Packages.gdal)</li>
    <li>gdalMinimal</li>
    <li>gdcm</li>
    <li>gmt</li>
    <li>grass</li>
    <li>haskellPackages.hgdal</li>
    <li>haskellPackages.hgdal.doc</li>
    <li>itk (itk_5)</li>
    <li>itk_5_2</li>
    <li>mapcache</li>
    <li>mapnik</li>
    <li>mapproxy</li>
    <li>mapproxy.dist</li>
    <li>mapserver</li>
    <li>mirtk</li>
    <li>mysql-workbench</li>
    <li>octavePackages.dicom</li>
    <li>octavePackages.mapping</li>
    <li>openems</li>
    <li>openorienteering-mapper</li>
    <li>osmscout-server</li>
    <li>paraview</li>
    <li>parmmg</li>
    <li>parmmg.dev</li>
    <li>pcl</li>
    <li>pdal</li>
    <li>perl538Packages.Tirex</li>
    <li>perl538Packages.Tirex.devdoc</li>
    <li>perlPackages.Tirex (perl540Packages.Tirex)</li>
    <li>perlPackages.Tirex.devdoc (perl540Packages.Tirex.devdoc)</li>
    <li>postgresql14Packages.postgis</li>
    <li>postgresql14Packages.postgis.doc</li>
    <li>postgresql15Packages.postgis</li>
    <li>postgresql15Packages.postgis.doc</li>
    <li>postgresql16Packages.postgis</li>
    <li>postgresql16Packages.postgis.doc</li>
    <li>postgresqlPackages.postgis (postgresql17Packages.postgis)</li>
    <li>postgresqlPackages.postgis.doc (postgresql17Packages.postgis.doc)</li>
    <li>postgresql18Packages.postgis</li>
    <li>postgresql18Packages.postgis.doc</li>
    <li>python312Packages.bsuite</li>
    <li>python312Packages.bsuite.dist</li>
    <li>python312Packages.cartopy</li>
    <li>python312Packages.cartopy.dist</li>
    <li>python312Packages.dicom2nifti</li>
    <li>python312Packages.dicom2nifti.dist</li>
    <li>python312Packages.django-bootstrap4</li>
    <li>python312Packages.django-bootstrap4.dist</li>
    <li>python312Packages.django-bootstrap5</li>
    <li>python312Packages.django-bootstrap5.dist</li>
    <li>python312Packages.drf-extra-fields</li>
    <li>python312Packages.drf-extra-fields.dist</li>
    <li>python312Packages.f3d</li>
    <li>python312Packages.f3d.man</li>
    <li>python312Packages.fast-simplification</li>
    <li>python312Packages.fast-simplification.dist</li>
    <li>python312Packages.fiona</li>
    <li>python312Packages.fiona.dist</li>
    <li>python312Packages.firedrake</li>
    <li>python312Packages.firedrake.dist</li>
    <li>python312Packages.folium</li>
    <li>python312Packages.folium.dist</li>
    <li>python312Packages.gdal</li>
    <li>python312Packages.gdcm</li>
    <li>python312Packages.geoarrow-pyarrow</li>
    <li>python312Packages.geoarrow-pyarrow.dist</li>
    <li>python312Packages.geodatasets</li>
    <li>python312Packages.geodatasets.dist</li>
    <li>python312Packages.geopandas</li>
    <li>python312Packages.geopandas.dist</li>
    <li>python312Packages.geoparquet</li>
    <li>python312Packages.geoparquet.dist</li>
    <li>python312Packages.gstools</li>
    <li>python312Packages.gstools.dist</li>
    <li>python312Packages.inequality</li>
    <li>python312Packages.inequality.dist</li>
    <li>python312Packages.itk</li>
    <li>python312Packages.libpysal</li>
    <li>python312Packages.libpysal.dist</li>
    <li>python312Packages.lida</li>
    <li>python312Packages.lida.dist</li>
    <li>python312Packages.mapclassify</li>
    <li>python312Packages.mapclassify.dist</li>
    <li>python312Packages.mayavi</li>
    <li>python312Packages.mayavi.dist</li>
    <li>python312Packages.momepy</li>
    <li>python312Packages.momepy.dist</li>
    <li>python312Packages.morecantile</li>
    <li>python312Packages.morecantile.dist</li>
    <li>python312Packages.msticpy</li>
    <li>python312Packages.msticpy.dist</li>
    <li>python312Packages.netbox-documents</li>
    <li>python312Packages.netbox-documents.dist</li>
    <li>python312Packages.odc-geo</li>
    <li>python312Packages.odc-geo.dist</li>
    <li>python312Packages.odc-loader</li>
    <li>python312Packages.odc-loader.dist</li>
    <li>python312Packages.odc-stac</li>
    <li>python312Packages.odc-stac.dist</li>
    <li>python312Packages.opencv4Full</li>
    <li>python312Packages.opencv4Full.cxxdev</li>
    <li>python312Packages.opencv4Full.package_tests</li>
    <li>python312Packages.osmnx</li>
    <li>python312Packages.osmnx.dist</li>
    <li>python312Packages.pandera</li>
    <li>python312Packages.pandera.dist</li>
    <li>python312Packages.plotnine</li>
    <li>python312Packages.plotnine.dist</li>
    <li>python312Packages.pygmt</li>
    <li>python312Packages.pygmt.dist</li>
    <li>python312Packages.pykrige</li>
    <li>python312Packages.pykrige.dist</li>
    <li>python312Packages.pymatgen</li>
    <li>python312Packages.pymatgen.dist</li>
    <li>python312Packages.pyogrio</li>
    <li>python312Packages.pyogrio.dist</li>
    <li>python312Packages.python-csxcad</li>
    <li>python312Packages.python-csxcad.dist</li>
    <li>python312Packages.python-openems</li>
    <li>python312Packages.python-openems.dist</li>
    <li>python312Packages.pyvista</li>
    <li>python312Packages.pyvista.dist</li>
    <li>python312Packages.rasterio</li>
    <li>python312Packages.rasterio.dist</li>
    <li>python312Packages.rio-stac</li>
    <li>python312Packages.rio-stac.dist</li>
    <li>python312Packages.rioxarray</li>
    <li>python312Packages.rioxarray.dist</li>
    <li>python312Packages.sfepy</li>
    <li>python312Packages.sfepy.dist</li>
    <li>python312Packages.shimmy</li>
    <li>python312Packages.shimmy.dist</li>
    <li>python312Packages.sumo</li>
    <li>python312Packages.sumo.dist</li>
    <li>python312Packages.vmas</li>
    <li>python312Packages.vmas.dist</li>
    <li>python312Packages.vtk</li>
    <li>python312Packages.wktutils</li>
    <li>python312Packages.wktutils.dist</li>
    <li>python313Packages.bsuite</li>
    <li>python313Packages.bsuite.dist</li>
    <li>python313Packages.cartopy</li>
    <li>python313Packages.cartopy.dist</li>
    <li>python313Packages.dicom2nifti</li>
    <li>python313Packages.dicom2nifti.dist</li>
    <li>python313Packages.django-bootstrap4</li>
    <li>python313Packages.django-bootstrap4.dist</li>
    <li>python313Packages.django-bootstrap5</li>
    <li>python313Packages.django-bootstrap5.dist</li>
    <li>python313Packages.drf-extra-fields</li>
    <li>python313Packages.drf-extra-fields.dist</li>
    <li>python313Packages.f3d</li>
    <li>python313Packages.f3d.man</li>
    <li>python313Packages.fast-simplification</li>
    <li>python313Packages.fast-simplification.dist</li>
    <li>python313Packages.fiona</li>
    <li>python313Packages.fiona.dist</li>
    <li>python313Packages.firedrake</li>
    <li>python313Packages.firedrake.dist</li>
    <li>python313Packages.folium</li>
    <li>python313Packages.folium.dist</li>
    <li>python313Packages.gdcm</li>
    <li>python313Packages.geoarrow-pyarrow</li>
    <li>python313Packages.geoarrow-pyarrow.dist</li>
    <li>python313Packages.geodatasets</li>
    <li>python313Packages.geodatasets.dist</li>
    <li>python313Packages.geopandas</li>
    <li>python313Packages.geopandas.dist</li>
    <li>python313Packages.geoparquet</li>
    <li>python313Packages.geoparquet.dist</li>
    <li>python313Packages.gstools</li>
    <li>python313Packages.gstools.dist</li>
    <li>python313Packages.inequality</li>
    <li>python313Packages.inequality.dist</li>
    <li>python313Packages.itk</li>
    <li>python313Packages.libpysal</li>
    <li>python313Packages.libpysal.dist</li>
    <li>python313Packages.lida</li>
    <li>python313Packages.lida.dist</li>
    <li>python313Packages.mapclassify</li>
    <li>python313Packages.mapclassify.dist</li>
    <li>python313Packages.mayavi</li>
    <li>python313Packages.mayavi.dist</li>
    <li>python313Packages.momepy</li>
    <li>python313Packages.momepy.dist</li>
    <li>python313Packages.morecantile</li>
    <li>python313Packages.morecantile.dist</li>
    <li>python313Packages.msticpy</li>
    <li>python313Packages.msticpy.dist</li>
    <li>python313Packages.netbox-documents</li>
    <li>python313Packages.netbox-documents.dist</li>
    <li>python313Packages.odc-geo</li>
    <li>python313Packages.odc-geo.dist</li>
    <li>python313Packages.odc-loader</li>
    <li>python313Packages.odc-loader.dist</li>
    <li>python313Packages.odc-stac</li>
    <li>python313Packages.odc-stac.dist</li>
    <li>python313Packages.opencv4Full</li>
    <li>python313Packages.opencv4Full.cxxdev</li>
    <li>python313Packages.opencv4Full.package_tests</li>
    <li>python313Packages.osmnx</li>
    <li>python313Packages.osmnx.dist</li>
    <li>python313Packages.pandera</li>
    <li>python313Packages.pandera.dist</li>
    <li>python313Packages.plotnine</li>
    <li>python313Packages.plotnine.dist</li>
    <li>python313Packages.pygmt</li>
    <li>python313Packages.pygmt.dist</li>
    <li>python313Packages.pykrige</li>
    <li>python313Packages.pykrige.dist</li>
    <li>python313Packages.pyogrio</li>
    <li>python313Packages.pyogrio.dist</li>
    <li>python313Packages.python-csxcad</li>
    <li>python313Packages.python-csxcad.dist</li>
    <li>python313Packages.pyvista</li>
    <li>python313Packages.pyvista.dist</li>
    <li>python313Packages.rasterio</li>
    <li>python313Packages.rasterio.dist</li>
    <li>python313Packages.rio-stac</li>
    <li>python313Packages.rio-stac.dist</li>
    <li>python313Packages.rioxarray</li>
    <li>python313Packages.rioxarray.dist</li>
    <li>python313Packages.shimmy</li>
    <li>python313Packages.shimmy.dist</li>
    <li>python313Packages.vmas</li>
    <li>python313Packages.vmas.dist</li>
    <li>python313Packages.vtk</li>
    <li>python313Packages.wktutils</li>
    <li>python313Packages.wktutils.dist</li>
    <li>pytrainer</li>
    <li>pytrainer.dist</li>
    <li>qcsxcad</li>
    <li>qcsxcad.dev</li>
    <li>qgis</li>
    <li>qgis-ltr</li>
    <li>qmapshack</li>
    <li>rtabmap</li>
    <li>saga</li>
    <li>supercell-wx</li>
    <li>survex</li>
    <li>therion</li>
    <li>tunnelx</li>
    <li>udig</li>
    <li>valhalla</li>
    <li>vpv</li>
    <li>vtk (vtk_9_5)</li>
    <li>vtk-full</li>
    <li>vtkWithQt6</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>ants</summary>
<pre>-- AverageImages 
-- simpleSynRegistration 
-- CompositeTransformUtil 
-- CreateDisplacementField 
-- ConvertTransformFile 
-- compareTwoTransforms 
-- SuperResolution 
-- TimeSCCAN 
-- TextureCooccurrenceFeatures 
-- TextureRunLengthFeatures 
-- ImageIntensityStatistics 
-- GetConnectedComponentsFeatureImages 
-- DeNrrd 
-- StackSlices 
-- antsSurf VTK::RenderingVolumeOpenGL2;VTK::CommonCore;VTK::CommonDataModel;VTK::IOGeometry;VTK::IOXML;VTK::IOLegacy;VTK::IOPLY;VTK::FiltersModeling;VTK::ImagingStencil;VTK::ImagingGeneral;VTK::RenderingAnnotation;VTK::FiltersExtraction
-- antsVol VTK::RenderingVolumeOpenGL2;VTK::CommonCore;VTK::CommonDataModel;VTK::IOGeometry;VTK::IOXML;VTK::IOLegacy;VTK::IOPLY;VTK::FiltersModeling;VTK::ImagingStencil;VTK::ImagingGeneral;VTK::RenderingAnnotation;VTK::FiltersExtraction
-- GetMeshAndTopology VTK::RenderingVolumeOpenGL2;VTK::CommonCore;VTK::CommonDataModel;VTK::IOGeometry;VTK::IOXML;VTK::IOLegacy;VTK::IOPLY;VTK::FiltersModeling;VTK::ImagingStencil;VTK::ImagingGeneral;VTK::RenderingAnnotation;VTK::FiltersExtraction
-- CheckTopology VTK::RenderingVolumeOpenGL2;VTK::CommonCore;VTK::CommonDataModel;VTK::IOGeometry;VTK::IOXML;VTK::IOLegacy;VTK::IOPLY;VTK::FiltersModeling;VTK::ImagingStencil;VTK::ImagingGeneral;VTK::RenderingAnnotation;VTK::FiltersExtraction
-- WarpVTKPolyDataMultiTransform VTK::RenderingVolumeOpenGL2;VTK::CommonCore;VTK::CommonDataModel;VTK::IOGeometry;VTK::IOXML;VTK::IOLegacy;VTK::IOPLY;VTK::FiltersModeling;VTK::ImagingStencil;VTK::ImagingGeneral;VTK::RenderingAnnotation;VTK::FiltersExtraction
-- Configuring incomplete, errors occurred!</pre>
</details>
<details>
<summary>apacheHttpdPackages.mod_tile</summary>
<pre>21/25 Test #19: status_and_dirty_webp ............   Passed    2.13 sec
      Start 20: remove_tile_webp
22/25 Test #20: remove_tile_webp .................   Passed    0.00 sec
      Start 23: stop_renderd
23/25 Test #23: stop_renderd .....................   Passed    0.02 sec
      Start 24: stop_httpd
24/25 Test #24: stop_httpd .......................   Passed    0.01 sec
      Start 25: clear_dirs
25/25 Test #25: clear_dirs .......................   Passed    0.02 sec

88% tests passed, 3 tests failed out of 25

Total Test time (real) = 213.62 sec

The following tests FAILED:
          6 - render_expired (Timeout)
          7 - render_list (Timeout)
          8 - render_old (Timeout)
Errors while running CTest
make: *** [Makefile:91: test] Error 8</pre>
</details>
<details>
<summary>cloudcompare</summary>
<pre>/build/source/libs/qCC_db/include/ccGLMatrixTpl.h: In instantiation of &#x27;bool ccGLMatrixTpl&lt;T&gt;::toAsciiFile(QString, int) const [with T = float]&#x27;:
/build/source/libs/qCC_db/include/ccGLMatrixTpl.h:343:15:   required from here
  343 |         virtual bool toAsciiFile(QString filename, int precision = 12) const
      |                      ^~~~~~~~~~~
/build/source/libs/qCC_db/include/ccGLMatrixTpl.h:354:117: warning: &#x27;QTextStream&amp; QTextStreamFunctions::endl(QTextStream&amp;)&#x27; is deprecated: Use Qt::endl [8;;https://gcc.gnu.org/onlinedocs/gcc-14.3.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  354 |                         stream &lt;&lt; m_mat[i] &lt;&lt; &quot; &quot; &lt;&lt; m_mat[i + 4] &lt;&lt; &quot; &quot; &lt;&lt; m_mat[i + 8] &lt;&lt; &quot; &quot; &lt;&lt; m_mat[i + 12] &lt;&lt; endl;
      |                                                                                                                     ^~~~
/nix/store/5f0nxzggzzqlfyn0gfcj83sna1igmsl3-qtbase-5.15.18-dev/include/QtCore/qtextstream.h:293:75: note: declared here
  293 | Q_CORE_EXPORT QT_DEPRECATED_VERSION_X(5, 15, &quot;Use Qt::endl&quot;) QTextStream &amp;endl(QTextStream &amp;s);
      |                                                                           ^~~~
/build/source/libs/qCC_db/include/ccGLMatrixTpl.h:354:114: warning: &#x27;QTextStream&amp; QTextStreamFunctions::endl(QTextStream&amp;)&#x27; is deprecated: Use Qt::endl [8;;https://gcc.gnu.org/onlinedocs/gcc-14.3.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  354 |                         stream &lt;&lt; m_mat[i] &lt;&lt; &quot; &quot; &lt;&lt; m_mat[i + 4] &lt;&lt; &quot; &quot; &lt;&lt; m_mat[i + 8] &lt;&lt; &quot; &quot; &lt;&lt; m_mat[i + 12] &lt;&lt; endl;
      |                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~
/nix/store/5f0nxzggzzqlfyn0gfcj83sna1igmsl3-qtbase-5.15.18-dev/include/QtCore/qtextstream.h:293:75: note: declared here
  293 | Q_CORE_EXPORT QT_DEPRECATED_VERSION_X(5, 15, &quot;Use Qt::endl&quot;) QTextStream &amp;endl(QTextStream &amp;s);
      |                                                                           ^~~~
[ 77%] Built target ccViewer
[ 78%] Linking CXX executable CloudCompare
Read from remote host gottlob: Connection reset by peer
client_loop: send disconnect: Broken pipe</pre>
</details>
<details>
<summary>gplates</summary>
<pre>[ 43%] Building CXX object src/CMakeFiles/gplates-lib.dir/gui/RasterColourPalette.cc.o
[ 43%] Building CXX object src/CMakeFiles/gplates-lib.dir/gui/SceneLightingParameters.cc.o
[ 43%] Building CXX object src/CMakeFiles/gplates-lib.dir/gui/SessionMenu.cc.o
[ 43%] Building CXX object src/CMakeFiles/gplates-lib.dir/gui/SimpleGlobeOrientation.cc.o
[ 43%] Building CXX object src/CMakeFiles/gplates-lib.dir/gui/SingleColourScheme.cc.o
[ 43%] Building CXX object src/CMakeFiles/gplates-lib.dir/gui/SmallCircleCanvasToolWorkflow.cc.o
[ 43%] Building CXX object src/CMakeFiles/gplates-lib.dir/gui/SphericalGrid.cc.o
[ 43%] Building CXX object src/CMakeFiles/gplates-lib.dir/gui/Stars.cc.o
[ 43%] Building CXX object src/CMakeFiles/gplates-lib.dir/gui/Symbol.cc.o
[ 44%] Building CXX object src/CMakeFiles/gplates-lib.dir/gui/TextOverlay.cc.o
[ 44%] Building CXX object src/CMakeFiles/gplates-lib.dir/gui/TextOverlaySettings.cc.o
[ 44%] Building CXX object src/CMakeFiles/gplates-lib.dir/gui/TopologyCanvasToolWorkflow.cc.o
[ 44%] Building CXX object src/CMakeFiles/gplates-lib.dir/gui/TopologySectionsContainer.cc.o
[ 44%] Building CXX object src/CMakeFiles/gplates-lib.dir/gui/TopologySectionsTable.cc.o
g++: fatal error: Killed signal terminated program cc1plus
compilation terminated.
make[2]: *** [src/CMakeFiles/gplates-lib.dir/build.make:6012: src/CMakeFiles/gplates-lib.dir/gui/PythonManager.cc.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:600: src/CMakeFiles/gplates-lib.dir/all] Error 2
make: *** [Makefile:156: all] Error 2</pre>
</details>
<details>
<summary>merkaartor</summary>
<pre>      |                                    |
      |                                    const OGRSpatialReference*
/build/source/src/ImportExport/ImportExportGdal.cpp:393:71: error: invalid conversion from &#x27;const OGRFieldDefn*&#x27; to &#x27;OGRFieldDefn*&#x27; [8;;https://gcc.gnu.org/onlinedocs/gcc-14.3.0/gcc/Warning-Options.html#index-fpermissive-fpermissive8;;]
  393 |                         OGRFieldDefn  *fd = poFeature-&gt;GetFieldDefnRef(i);
      |                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~^~~
      |                                                                       |
      |                                                                       const OGRFieldDefn*
[ 60%] Building CXX object CMakeFiles/merkaartor.dir/src/common/MapView.cpp.o
[ 60%] Building CXX object CMakeFiles/merkaartor.dir/src/common/Coord.cpp.o
/build/source/src/ImportExport/ImportExportGdal.cpp: In member function &#x27;virtual bool ImportExportGdal::export_(const QList&lt;Feature*&gt;&amp;)&#x27;:
/build/source/src/ImportExport/ImportExportGdal.cpp:158:25: warning: ignoring return value of function declared with attribute &#x27;warn_unused_result&#x27; [8;;https://gcc.gnu.org/onlinedocs/gcc-14.3.0/gcc/Warning-Options.html#index-Wno-unused-result-Wunused-result8;;]
  158 |     poLayer-&gt;CreateField( &amp;oField );
      |     ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~
/build/source/src/ImportExport/ImportExportGdal.cpp:160:25: warning: ignoring return value of function declared with attribute &#x27;warn_unused_result&#x27; [8;;https://gcc.gnu.org/onlinedocs/gcc-14.3.0/gcc/Warning-Options.html#index-Wno-unused-result-Wunused-result8;;]
  160 |     poLayer-&gt;CreateField( &amp;oField );
      |     ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~
make[2]: *** [CMakeFiles/merkaartor.dir/build.make:1328: CMakeFiles/merkaartor.dir/src/ImportExport/ImportExportGdal.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:148: CMakeFiles/merkaartor.dir/all] Error 2
make: *** [Makefile:166: all] Error 2</pre>
</details>
<details>
<summary>otb</summary>
<pre>/build/source/Modules/Core/Adapters/GdalAdapters/include/otbOGRFieldWrapper.h:63:27: note:   initializing argument 1 of &#x27;otb::ogr::FieldDefn::FieldDefn(OGRFieldDefn&amp;)&#x27;
   63 |   FieldDefn(OGRFieldDefn&amp; definition) : m_Definition(&amp;definition)
      |             ~~~~~~~~~~~~~~^~~~~~~~~~
make[2]: *** [Modules/Core/Adapters/GdalAdapters/src/CMakeFiles/OTBGdalAdapters.dir/build.make:79: Modules/Core/Adapters/GdalAdapters/src/CMakeFiles/OTBGdalAdapters.dir/otbOGRFieldWrapper.cxx.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:7965: Modules/Core/Adapters/GdalAdapters/src/CMakeFiles/OTBGdalAdapters.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 19%] Building CXX object Modules/Core/Adapters/CurlAdapters/src/CMakeFiles/OTBCurlAdapters.dir/otbCurlHelperInterface.cxx.o
[ 19%] Built target OTBCommonHeaderTest2
[ 19%] Linking C shared library ../../../../lib/libotb6S.so
[ 19%] Linking CXX executable ../../../bin/OTBCommonHeaderTest1
[ 19%] Built target otb6S
[ 19%] Linking CXX executable ../../../bin/OTBITKHeaderTest1
[ 19%] Built target OTBObjectListHeaderTest1
[ 19%] Linking CXX shared library ../../../../../lib/libOTBCurlAdapters.so
[ 19%] Built target OTBCommonHeaderTest1
[ 19%] Built target OTBCurlAdapters
[ 19%] Built target OTBITKHeaderTest1
[ 19%] Linking CXX executable ../../../../bin/OTBBoostAdaptersHeaderTest1
[ 19%] Built target OTBBoostAdaptersHeaderTest1
make: *** [Makefile:166: all] Error 2</pre>
</details>
<details>
<summary>simpleitk</summary>
<pre>[ 99%] Building CXX object Code/BasicFilters/src/CMakeFiles/SimpleITK_ITKImageIntensity.dir/sitkSqrtImageFilter.cxx.o
[ 99%] Building CXX object Code/BasicFilters/src/CMakeFiles/SimpleITK_ITKImageIntensity.dir/sitkSquareImageFilter.cxx.o
[ 99%] Building CXX object Code/BasicFilters/src/CMakeFiles/SimpleITK_ITKImageIntensity.dir/sitkSubtractImageFilter.cxx.o
[ 99%] Building CXX object Code/BasicFilters/src/CMakeFiles/SimpleITK_ITKImageIntensity.dir/sitkTanImageFilter.cxx.o
[ 99%] Building CXX object Code/BasicFilters/src/CMakeFiles/SimpleITK_ITKImageIntensity.dir/sitkTernaryAddImageFilter.cxx.o
[100%] Building CXX object Code/BasicFilters/src/CMakeFiles/SimpleITK_ITKImageIntensity.dir/sitkTernaryMagnitudeImageFilter.cxx.o
[100%] Building CXX object Code/BasicFilters/src/CMakeFiles/SimpleITK_ITKImageIntensity.dir/sitkTernaryMagnitudeSquaredImageFilter.cxx.o
[100%] Building CXX object Code/BasicFilters/src/CMakeFiles/SimpleITK_ITKImageIntensity.dir/sitkVectorIndexSelectionCastImageFilter.cxx.o
[100%] Building CXX object Code/BasicFilters/src/CMakeFiles/SimpleITK_ITKImageIntensity.dir/sitkVectorMagnitudeImageFilter.cxx.o
[100%] Building CXX object Code/BasicFilters/src/CMakeFiles/SimpleITK_ITKImageIntensity.dir/sitkXorImageFilter.cxx.o
[100%] Linking CXX static library ../../../lib/libSimpleITK_ITKImageIntensity-2.5.a
[100%] Built target SimpleITK_ITKImageIntensity
[100%] Building CXX object Code/BasicFilters/src/CMakeFiles/SimpleITKBasicFilters1.dir/sitkAdditionalProcedures.cxx.o
[100%] Linking CXX static library ../../../lib/libSimpleITKBasicFilters1-2.5.a
[100%] Built target SimpleITKBasicFilters1
[100%] Swig source to generate LUA wrapping
[100%] Building CXX object Wrapping/Lua/CMakeFiles/SimpleITKLuaModule_LUA.dir/SimpleITKLUA_wrap.cxx.o
[100%] Linking CXX shared module lib/SimpleITK.so
Read from remote host gottlob: Connection reset by peer
client_loop: send disconnect: Broken pipe</pre>
</details>
<details>
<summary>sumo</summary>
<pre>[ 53%] Building CXX object src/guisim/CMakeFiles/guisim.dir/GUIVehicleControl.cpp.o
[ 53%] Building CXX object src/netimport/vissim/typeloader/CMakeFiles/netimport_vissim_typeloader.dir/NIVissimSingleTypeParser_Streckendefinition.cpp.o
[ 53%] Building CXX object src/netimport/vissim/typeloader/CMakeFiles/netimport_vissim_typeloader.dir/NIVissimSingleTypeParser_Streckentypdefinition.cpp.o
[ 53%] Building CXX object src/netimport/vissim/typeloader/CMakeFiles/netimport_vissim_typeloader.dir/NIVissimSingleTypeParser_VWunschentscheidungsdefinition.cpp.o
[ 53%] Building CXX object src/netimport/vissim/typeloader/CMakeFiles/netimport_vissim_typeloader.dir/NIVissimSingleTypeParser_TEAPACDefinition.cpp.o
[ 53%] Building CXX object src/netimport/vissim/typeloader/CMakeFiles/netimport_vissim_typeloader.dir/NIVissimSingleTypeParser_Verbindungsdefinition.cpp.o
[ 53%] Building CXX object src/guisim/CMakeFiles/guisim.dir/GUITransportableControl.cpp.o
[ 53%] Building CXX object src/netimport/vissim/typeloader/CMakeFiles/netimport_vissim_typeloader.dir/NIVissimSingleTypeParser_Verkehrszusammensetzungsdefinition.cpp.o
[ 53%] Building CXX object src/netimport/vissim/typeloader/CMakeFiles/netimport_vissim_typeloader.dir/NIVissimSingleTypeParser_Verlustzeitmessungsdefinition.cpp.o
[ 53%] Building CXX object src/netimport/vissim/typeloader/CMakeFiles/netimport_vissim_typeloader.dir/NIVissimSingleTypeParser_Zeitenverteilungsdefinition.cpp.o
[ 54%] Building CXX object src/netimport/vissim/typeloader/CMakeFiles/netimport_vissim_typeloader.dir/NIVissimSingleTypeParser_Zeitschrittfaktor.cpp.o
[ 54%] Building CXX object src/netimport/vissim/typeloader/CMakeFiles/netimport_vissim_typeloader.dir/NIVissimSingleTypeParser_Zuflussdefinition.cpp.o
[ 54%] Building CXX object src/netimport/vissim/typeloader/CMakeFiles/netimport_vissim_typeloader.dir/NIVissimSingleTypeParser_Zusammensetzungsdefinition.cpp.o
[ 54%] Building CXX object src/netimport/vissim/typeloader/CMakeFiles/netimport_vissim_typeloader.dir/NIVissimSingleTypeParser__XKurvedefinition.cpp.o
[ 54%] Building CXX object src/netimport/vissim/typeloader/CMakeFiles/netimport_vissim_typeloader.dir/NIVissimSingleTypeParser__XVerteilungsdefinition.cpp.o
[ 54%] Linking CXX static library libguisim.a
[ 54%] Built target guisim
[ 54%] Linking CXX static library libnetimport_vissim_typeloader.a
[ 54%] Built target netimport_vissim_typeloader
make: *** [Makefile:146: all] Error 2</pre>
</details>
<details>
<summary>vtk-dicom</summary>
<pre>Call Stack (most recent call first):
  /nix/store/3kcibf22q0x7kg50gwy1v8iv1rkrjyp8-adios2-2.11.0/lib/cmake/adios2/adios2-config.cmake:30 (include)
  /nix/store/isazd08x76rivw5awmg34gkxkd2jmz12-vtk-9.5.2/lib/cmake/vtk/VTK-vtk-module-find-packages.cmake:403 (find_package)
  /nix/store/isazd08x76rivw5awmg34gkxkd2jmz12-vtk-9.5.2/lib/cmake/vtk/vtk-config.cmake:171 (include)
  /nix/store/r6z8c95glpi6pbzcxwqfrqrb4m0wf1lz-gdcm-3.2.2/lib/gdcm-3.2/UseGDCM.cmake:24 (find_package)
  Source/CMakeLists.txt:166 (include)


CMake Error at /nix/store/3kcibf22q0x7kg50gwy1v8iv1rkrjyp8-adios2-2.11.0/lib/cmake/adios2/adios2-config-common.cmake:264 (target_link_libraries):
  Cannot specify link libraries for target &quot;adios2::cxx11&quot; which is not built
  by this project.
Call Stack (most recent call first):
  /nix/store/3kcibf22q0x7kg50gwy1v8iv1rkrjyp8-adios2-2.11.0/lib/cmake/adios2/adios2-config.cmake:30 (include)
  /nix/store/isazd08x76rivw5awmg34gkxkd2jmz12-vtk-9.5.2/lib/cmake/vtk/VTK-vtk-module-find-packages.cmake:403 (find_package)
  /nix/store/isazd08x76rivw5awmg34gkxkd2jmz12-vtk-9.5.2/lib/cmake/vtk/vtk-config.cmake:171 (include)
  /nix/store/r6z8c95glpi6pbzcxwqfrqrb4m0wf1lz-gdcm-3.2.2/lib/gdcm-3.2/UseGDCM.cmake:24 (find_package)
  Source/CMakeLists.txt:166 (include)


-- Configuring incomplete, errors occurred!</pre>
</details>
</details>

--- a/pkgs/by-name/me/merkaartor/package.nix
+++ b/pkgs/by-name/me/merkaartor/package.nix
@@ -2,6 +2,8 @@
   lib,
   stdenv,
   fetchFromGitHub,
+  fetchpatch,
+
   cmake,
   pkg-config,
   gdal,
@@ -28,6 +30,15 @@ stdenv.mkDerivation rec {
     tag = version;
     hash = "sha256-oxLGhIE1qJ9+GOztD1HvrLGRGVO3gyy7Rc6CyzKTFec=";
   };
+
+  patches = [
+    # Fix for GDAL 3.12
+    # https://github.com/openstreetmap/merkaartor/pull/316
+    (fetchpatch {
+      url = "https://github.com/openstreetmap/merkaartor/commit/28cca84e9f5db0aaba87c2084ed32f9677598823.diff";
+      hash = "sha256-so0La5djYhWF6NqLpShWa3vGl5A2jkS3Xwg5Pe1yse4=";
+    })
+  ];
 
   nativeBuildInputs = [
     cmake

--- a/pkgs/by-name/me/merkaartor/package.nix
+++ b/pkgs/by-name/me/merkaartor/package.nix
@@ -82,7 +82,7 @@ stdenv.mkDerivation rec {
     homepage = "https://merkaartor.be/";
     license = lib.licenses.gpl2Plus;
     mainProgram = "merkaartor";
-    maintainers = with lib.maintainers; [ sikmir ];
+    teams = [ lib.teams.geospatial ];
     platforms = lib.platforms.unix;
   };
 }

--- a/pkgs/by-name/su/sumo/package.nix
+++ b/pkgs/by-name/su/sumo/package.nix
@@ -32,13 +32,13 @@
 
 stdenv.mkDerivation rec {
   pname = "sumo";
-  version = "1.24.0";
+  version = "1.25.0";
 
   src = fetchFromGitHub {
-    owner = "eclipse";
+    owner = "eclipse-sumo";
     repo = "sumo";
     tag = "v${lib.replaceStrings [ "." ] [ "_" ] version}";
-    hash = "sha256-xf7/hUJpl+XmXx5MmFzYu2geFNe7JVaxDrraoqLrSuk=";
+    hash = "sha256-rUa5DpoBfnviSEmzPEiVnZU0KGavAIOFoysQ74uTll0=";
     fetchSubmodules = true;
   };
 

